usr/lib/python3/dist-packages/linuxmusterLinuxclient7/printers.py

```diff
@@ -73,16 +73,20 @@ def translateSambaToIpp(networkPath):
 # --------------------
 
 def _installPrinter(username, networkPath, name):
-    logging.debug("Installing Printer {0} on {1}".format(name, networkPath))
-    installCommand = ["lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
+    logging.info("_installPrinter {0} on {1}".format(name, networkPath))
+    installCommand = ["timeout", "3", "lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
     logging.debug("* running '{}'".format(" ".join(installCommand)))
-
-    if not subprocess.call(installCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE) == 0:
-        logging.fatal(f"* Error installing printer {name} on {networkPath}!\n")
-        return False
-
-    logging.debug("* Success!")
-    return True
+    p = subprocess.Popen(installCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    output, error = p.communicate()
+    if p.returncode  == 0:
+        logging.debug("* Success _installPrinter!")
+        return True
+    elif p.returncode == 124:
+        logging.debug("* Error Timeout lpadmin")
+    else:
+        logging.debug(f"* Error installing printer {name} on {networkPath}!\n")
```

Comment:
Please change this to logging.fatal. Otherwise no error popup will be shown.
```suggestion
        logging.fatal(f"* Error installing printer {name} on {networkPath}!\n")
```